x-pack/plugins/apm/server/routes/transaction_groups.ts

```diff
@@ -75,23 +76,35 @@ export const transactionGroupsChartsRoute = createRoute(() => ({
       transactionType,
       transactionName,
       uiFilters: uiFiltersJson,
```

Comment:
It's not json but actually a string

```suggestion
      uiFilters: uiFiltersStringified,
```

Reply:
so, it's JSON? 😛

Reply:
Oh, that's true 🤦

Reply:
I still think it would be more clear if it was called `uiFiltersSerialized` or `uiFiltersEncoded`.